app/collections/playlists.js

```diff
@@ -1,9 +1,47 @@
 import Backbone from 'backbone';
 import Playlist from '../models/playlist';
+import cozysdk from 'cozysdk-client';
 
 
 const Playlists = Backbone.Collection.extend({
-    model: Playlist
+   	
+   	model: Playlist,
+
+    initialize: function() {
+        this.on('add', this.onAdd, this);
+    },
+
+    onAdd: function(playlist) {
+        if (!playlist.get('_id')) {
+            playlist.save()
+        }
+    },
+    
+    comparator: function (collection) {
```

Comment:
2 misuses here:

1. what's passed to the `sortBy` comparator is a `model`, not a `collection`
2. if you just want to sort by an attribute in the model, just pass a string as comparator: `comparator: 'title'`